Loop/Managers/DeviceDataManager.swift

```diff
@@ -1339,69 +1340,70 @@ extension DeviceDataManager {
         do {
             command = try RemoteCommand.createRemoteCommand(notification: notification, allowedPresets: loopManager.settings.overridePresets).get()
         } catch {
-            log.error("Remote Notification Error: %{public}@", String(describing: error))
+            log.error("Remote Notification: Parse Error: %{public}@", String(describing: error))
             return
         }
 
         switch command {
             
         case .temporaryScheduleOverride(let override):
-            log.default("Enacting remote temporary override: %{public}@", String(describing: override))
+            log.default("Remote Notification: Enacting temporary override: %{public}@", String(describing: override))
             loopManager.mutateSettings { settings in settings.scheduleOverride = override }
         case .cancelTemporaryOverride:
-            log.default("Canceling temporary override from remote command")
+            log.default("Remote Notification: Canceling temporary override")
             loopManager.mutateSettings { settings in settings.scheduleOverride = nil }
         case .bolusEntry(let bolusAmount):
-            log.default("Enacting remote bolus entry: %{public}@", String(describing: bolusAmount))
+            log.default("Remote Notification: Enacting bolus entry: %{public}@", String(describing: bolusAmount))
             
             //Remote bolus requires validation from its remote source
             guard remoteDataServicesManager.validatePushNotificationSource(notification) else {
                 NotificationManager.sendRemoteBolusFailureNotification(for: RemoteCommandError.invalidOTP, amount: bolusAmount)
```

Comment:
This call above to `validatePushNotificationSource` would really benefit from throwing/returning an error rather than a bool result. That would let us report if the OTP was invalid vs the user doesn't have OTP setup on the Looper's device. Something to look at in a refactor.